flock_unix.go

```diff
@@ -85,7 +85,11 @@ func (f *Flock) lock(locked *bool, flag int) error {
 func (f *Flock) Unlock() error {
```

Comment:
Can you please move this function out to `flock.go` and remove it from `flock_unix.go` and `flock_windows.go`? With having the unexported `unlockInternal` method, we don't need the duplicated `Unlock` anymore.